tests/framework/user_guide/ForwardSamplingStrategies/gold/RunDir/SparseGrid/rom_output.xml

```diff
@@ -1,2701 +1,2706 @@
-        <sigma-A.sigma-D>0.0</sigma-A.sigma-D>
-        <sigma-B.decay-B>0.0</sigma-B.decay-B>
-        <sigma-B.sigma-D>0.0</sigma-B.sigma-D>
-        <decay-A.decay-B>0.0</decay-A.decay-B>
-        <sigma-A.decay-A>0.0</sigma-A.decay-A>
-        <decay-C.decay-D>0.0</decay-C.decay-D>
-        <sigma-B.decay-A>0.0</sigma-B.decay-A>
-        <sigma-C.decay-D>0.0</sigma-C.decay-D>
-        <sigma-A.sigma-C>0.0</sigma-A.sigma-C>
-        <sigma-C.sigma-D>0.0</sigma-C.sigma-D>
-        <sigma-B.decay-D>0.0</sigma-B.decay-D>
-        <sigma-C.decay-A>0.0</sigma-C.decay-A>
-        <sigma-A.decay-B>0.0</sigma-A.decay-B>
-        <decay-A.decay-D>0.0</decay-A.decay-D>
-        <sigma-A.sigma-B>0.0</sigma-A.sigma-B>
-        <sigma-D.decay-A>0.0</sigma-D.decay-A>
-        <sigma-C.decay-B>0.0</sigma-C.decay-B>
-        <sigma-A.decay-C>0.0</sigma-A.decay-C>
-      </sobolIndices>
-      <sobolTotalIndices>
-        <decay-B>0.99872819089</decay-B>
-        <decay-C>0.00127654379272</decay-C>
-        <decay-B.decay-C>4.73470764302e-06</decay-B.decay-C>
-        <sigma-C>2.08961802285e-18</sigma-C>
-        <decay-D>2.08273003704e-18</decay-D>
-        <sigma-A>2.07789676572e-18</sigma-A>
-        <decay-A>2.07177004661e-18</decay-A>
-        <sigma-D>2.06991530676e-18</sigma-D>
-        <sigma-B>2.06194927854e-18</sigma-B>
-        <sigma-D.decay-C>1.03826841205e-27</sigma-D.decay-C>
-        <sigma-D.decay-B>1.03826841205e-27</sigma-D.decay-B>
-        <sigma-B.decay-C>0.0</sigma-B.decay-C>
-        <sigma-A.decay-D>0.0</sigma-A.decay-D>
-        <sigma-B.sigma-C>0.0</sigma-B.sigma-C>
-        <sigma-C.decay-C>0.0</sigma-C.decay-C>
-        <sigma-D.decay-D>0.0</sigma-D.decay-D>
-        <decay-A.decay-C>0.0</decay-A.decay-C>
-        <decay-B.decay-D>0.0</decay-B.decay-D>
-        <sigma-A.sigma-D>0.0</sigma-A.sigma-D>
-        <sigma-B.decay-B>0.0</sigma-B.decay-B>
-        <sigma-B.sigma-D>0.0</sigma-B.sigma-D>
-        <decay-A.decay-B>0.0</decay-A.decay-B>
-        <sigma-A.decay-A>0.0</sigma-A.decay-A>
-        <decay-C.decay-D>0.0</decay-C.decay-D>
-        <sigma-B.decay-A>0.0</sigma-B.decay-A>
-        <sigma-C.decay-D>0.0</sigma-C.decay-D>
-        <sigma-A.sigma-C>0.0</sigma-A.sigma-C>
-        <sigma-C.sigma-D>0.0</sigma-C.sigma-D>
-        <sigma-B.decay-D>0.0</sigma-B.decay-D>
-        <sigma-C.decay-A>0.0</sigma-C.decay-A>
-        <sigma-A.decay-B>0.0</sigma-A.decay-B>
-        <decay-A.decay-D>0.0</decay-A.decay-D>
-        <sigma-A.sigma-B>0.0</sigma-A.sigma-B>
-        <sigma-D.decay-A>0.0</sigma-D.decay-A>
-        <sigma-C.decay-B>0.0</sigma-C.decay-B>
-        <sigma-A.decay-C>0.0</sigma-A.decay-C>
-      </sobolTotalIndices>
-    </D>
-  </time>
-
-  <time value="17280000.0">
-    <A>
-      <mean>0.925209005978</mean>
-      <expectedValue>0.925209005978</expectedValue>
-      <variance>4.17482012618e-05</variance>
-      <samples>145</samples>
-      <polyCoeffs>
-        <inputVariables>sigma-A,sigma-B,sigma-C,sig
```

Comment:
This seems like a pretty, uh, simplistic example with not even a linear contribution to the ROM. Not necessarily part of this merge request, but maybe a better model could be chosen so we're actually testing something nonzero.

Reply:
This is a test in the user guide, hopefully we will get updated soon.